src/System.Net.WebSockets.Client/tests/WebSocketHelper.cs

```diff
@@ -67,27 +67,47 @@ public static class WebSocketHelper
             ITestOutputHelper output,
             TimeSpan keepAliveInterval = default(TimeSpan))
         {
-            var cws = new ClientWebSocket();
-            if (keepAliveInterval.TotalSeconds > 0)
-            {
-                cws.Options.KeepAliveInterval = keepAliveInterval;
-            }
+            const int MaxTries = 5;
+            int BetweenTryDelaySeconds = 1000;
```

Comment:
Yup, fixed, thanks.  I initially had it as a const, hence the name, but then changed it when I added the backoff and neglected to rename as well.